src/Message/AuthorizeRequest.php

```diff
@@ -35,19 +35,24 @@ public function getTransactionIndicator()
     public function getData()
     {
         $res = [
-            'transaction_id'        => $this->getTransactionId(),
-            'amount'                => $this->getAmount(),
-            'currency'              => $this->getCurrency(),
-            'description'           => $this->getDescription(),
-            'success_url'           => $this->getReturnUrl(),
-            'cancel_url'            => $this->getReturnUrl(),
-            'error_url'             => $this->getReturnUrl(),
-            'callback_url'          => $this->getNotifyUrl(),
-            'customer'              => $this->getCustomer(),
+            'transaction_id' => $this->getTransactionId(),
+            'amount' => $this->getAmount(),
+            'currency' => $this->getCurrency(),
+            'description' => $this->getDescription(),
+            'success_url' => $this->getReturnUrl(),
+            'cancel_url' => $this->getReturnUrl(),
+            'error_url' => $this->getReturnUrl(),
+            'callback_url' => $this->getNotifyUrl(),
+            'customer' => $this->getCustomer(),
             'transaction_indicator' => $this->getTransactionIndicator() ?: Preauthorize::TRANSACTION_INDICATOR_SINGLE,
+            'extra_data' => [
+                '3ds:authenticationIndicator' => $this->get3DSAuthenticationIndicator(),
```

Comment:
In order to have consistency we can make this lowecase get3**DS**AuthenticationIndicator()